### PR TITLE
TS-4268: Verbose OCSP notes in syslog

### PR DESCRIPTION
**Pull request by randall**

### CODE REVIEW

**Comment by jpeach** [comment]
[approve ci]

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/608/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/504/ for details.

File: iocore/net/OCSPStapling.cc [hunk 1]
```diff
@@ -382,9 +382,11 @@ ocsp_update()
         if (cinf->resp_derlen == 0 || cinf->is_expire || cinf->expire_time < current_time) {
           ink_mutex_release(&cinf->stapling_mutex);
           if (stapling_refresh_response(cinf, &resp)) {
-            Note("Success to refresh OCSP response for 1 certificate.");
+            Debug("ssl_ocsp", "Successful OCSP refresh response for 1 certificate");
+            SSL_INCREMENT_DYN_STAT(ssl_ocsp_refreshed_cert_stat);
           } else {
-            Note("Fail to refresh OCSP response for 1 certificate.");
+            Debug("ssl_ocsp", "Failed to refresh OCSP for 1 certificate");
```
Comment (abh) [inline iocore/net/OCSPStapling.cc]: It'd be nice to keep the failures in the non-debug logs, I think. Nicer if it included some information about the cert that couldn't be updated.

**Pull request merged**